services/webhook/feishu.go

```diff
@@ -97,23 +97,40 @@ func (f *FeishuPayload) Push(p *api.PushPayload) (api.Payloader, error) {
 
 // Issue implements PayloadConvertor Issue method
 func (f *FeishuPayload) Issue(p *api.IssuePayload) (api.Payloader, error) {
-	text, issueTitle, attachmentText, _ := getIssuesPayloadInfo(p, noneLinkFormatter, true)
-
-	return newFeishuTextPayload(issueTitle + "\r\n" + text + "\r\n\r\n" + attachmentText), nil
+	title, link, by, operator, result, assignees := getIssuesInfo(p)
+	var res api.Payloader
+	if assignees != "" {
+		if p.Action == api.HookIssueAssigned || p.Action == api.HookIssueUnassigned || p.Action == api.HookIssueMilestoned {
+			res = newFeishuTextPayload(title + "\r\n" + link + "\r\n" + by + "\r\n" + operator + "\r\n" + result + "\r\n" + assignees + "\r\n\r\n" + p.Issue.Body)
+		} else {
+			res = newFeishuTextPayload(title + "\r\n" + link + "\r\n" + by + "\r\n" + operator + "\r\n" + assignees + "\r\n\r\n" + p.Issue.Body)
+		}
+	} else {
+		res = newFeishuTextPayload(title + "\r\n" + link + "\r\n" + by + "\r\n" + operator + "\r\n\r\n" + p.Issue.Body)
```

Comment:
Please use `fmt.Sprintf`.
And please no `\r\n`, simply use `\n` if possible.

Reply:
Simply because this code was ugly previously does not mean it needs to stay ugly.

Reply:
fixed